x-pack/plugin/geoip-enterprise-downloader/src/test/java/org/elasticsearch/xpack/geoip/EnterpriseGeoIpDownloaderLicenseListenerTests.java

```diff
@@ -48,6 +52,8 @@
 public class EnterpriseGeoIpDownloaderLicenseListenerTests extends ESTestCase {
 
     private ThreadPool threadPool;
+    @FixForMultiProject
+    private final ProjectResolver projectResolver = TestProjectResolvers.DEFAULT_PROJECT_ONLY;
```

Comment:
Are you going to fix this in a follow-up PR soon, or are you using the default project ID because `licenseStateChanged()` is not going to be made project-aware anytime soon? If it's the former, this is fine, but if it's the latter, I don't think we need a `@FixForMultiProject`; we can either add `@NotMultiProjectCapable` as well or just leave the annotation out and add a comment.